packages/gatsby-source-nacelle/package.json

```diff
@@ -21,16 +21,19 @@
   "author": "Nacelle Inc. (getnacelle.com)",
   "license": "ISC",
   "dependencies": {
+    "@nacelle/client-js-sdk": "^3.1.0",
+    "@nacelle/contentful-preview-connector": "^0.1.5",
     "fs-extra": "^9.0.1",
-    "gatsby-graphql-source-toolkit": "^0.5.0",
-    "gatsby-source-filesystem": "^2.3.29"
+    "gatsby-graphql-source-toolkit": "^0.6.3",
+    "lodash.isobject": "^3.0.2",
+    "lodash.transform": "^4.6.0"
   },
   "peerDependencies": {
     "gatsby": "^2.0.32"
   },
   "devDependencies": {
-    "eslint": "^7.6.0",
-    "lint-staged": "^10.2.11"
+    "eslint": "^7.13.0",
```

Comment:
This ESLint version is > the version in the lerna package.json (which gets hoisted). This should be the same version that's in the root of the project (7.6.0)

Reply:
No problem! I'll change it back to `7.6.0`.